subsys/nfc/lib/platform.c

```diff
@@ -166,17 +240,28 @@ void nfc_platform_event_handler(nrfx_nfct_evt_t const *event)
 	case NRFX_NFCT_EVT_FIELD_DETECTED:
 		LOG_DBG("Field detected");
 
+#if !IS_ENABLED(CONFIG_SOC_SERIES_NRF54HX)
```

Comment:
Not sure if this is needed, @maje-emb did dummy clock driver for nRF54H purpose but I don't know if what is a status of it in context of NCS.

Reply:
Dummy clock driver for nRF54H not yet present in NCS.